CarAccidentScenario/RoadConditionsException.java

```diff
@@ -0,0 +1,8 @@
+package CarAccidentScenario;
+
+public class RoadConditionsException extends Exception {
```

Comment:
**RoadConditions** feel like a state rather than an error. Maybe you can rename it so the name suggests an error.

Reply:
Done